game_eggs/tycoon_games/openrct2/README.md

```diff
@@ -0,0 +1,88 @@
+                <img src="https://avatars.githubusercontent.com/u/4533989" width="50px;" alt=""/><br /><sub><b>Red-Thirten</b></sub>
+            </a>
+            <br />
+            <a href="https://github.com/parkervcp/eggs/commits?author=lilkingjr1" title="Codes">💻</a>
+            <a href="https://github.com/parkervcp/eggs/commits?author=lilkingjr1" title="Maintains">🔨</a>
+        </td>
+        <td align="center">
+            <a href="https://github.com/parkervcp">
+                <img src="https://avatars.githubusercontent.com/u/1207679" width="50px;" alt=""/><br /><sub><b>parkervcp</b></sub>
+            </a>
+            <br />
+            <a href="https://github.com/parkervcp/eggs/commits?author=parkervcp" title="Codes">💻</a>
+            <a href="https://github.com/parkervcp/eggs/commits?author=parkervcp" title="Contributor">💡</a>
+        </td>
+        <td align="center">
+            <a href="https://github.com/janisozaur">
+                <img src="https://avatars.githubusercontent.com/u/550290" width="50px;" alt=""/><br /><sub><b>janisozaur</b></sub>
+            </a>
+            <br />
+            <a href="https://github.com/OpenRCT2/OpenRCT2/commits?author=janisozaur" title="Codes">💻</a>
+            <a href="https://github.com/OpenRCT2/OpenRCT2/commits?author=janisozaur" title="Contributor">💡</a>
+        </td>
+    </tr>
+</table>
+<!-- markdownlint-enable -->
+<!-- prettier-ignore-end -->
+
+___
+### Game Description
+From OpenRCT2's [Website](https://openrct2.org/):
+> OpenRCT2 is an open-source re-implementation of RollerCoaster Tycoon 2 (RCT2), expanding the game with new features, fixing bugs and raising game limits. The gameplay revolves around building and maintaining an amusement park containing attractions, shops and facilities. The player must try to make a profit and maintain a good park reputation whilst keeping the guests happy. OpenRCT2 allows for both scenario and sandbox play. Scenarios require the player to complete a certain objective in a set time limit whilst sandbox allows the player to build a more flexible park with optionally no restrictions or finance.
+___
+### Server Ports
+| Port | Default (TCP) |
+|---------|---------|
+| Game | 11753 |
+
+___
+### Installation/System Requirements
+
+|  | Minimum |
+|---------|---------|
+| Processor | Recent x86/64 (AMD/Intel) processor. No ARM support. |
+| RAM | 256-512 MiB (depends on # of clients & park size) |
+| Storage | 80 MB |
+| RCT2 Files | **Not required** |
+
+___
+### Save Files
+Save files and autosaves are located in this directory:
+```
+/home/container/ServerData/save/
+```
+If the "Load Latest Autosave" startup variable is enabled, and the `autosave/` directory exists, the server will load the latest autosave instead of the specified Save File.
+___
+### Console Commands
+As of OpenRCT2 v0.3.5.1, entering commands via Pterodactyl do nothing. Hopefully this will be resolved in a future OpenRCT2 update.
```

Comment:
The wording here suggest OpenRCT2, but there isn't one linked and I haven't seen enough investigation to put the blame on us. Can you please:
1. try running `cat` as your docker startup command and verifying interactivity
2. if that works, file a bug with our project and link it here

Reply:
Pterodactyl does not support interactive programs at this time. It only runs a specific application. Ref: https://github.com/pterodactyl/panel/issues/3712

Reply:
> The wording here suggest OpenRCT2, but there isn't one linked and I haven't seen enough investigation to put the blame on us.

My apologies; this is a lack of Pterodactyl understanding on my part. I will fix this wording in my next commit.

Reply:
Fixed in commit 7dcbd35.